extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -164,11 +181,33 @@ export class AmpAnimation extends AMP.BaseElement {
     }
   }
 
+  /** @private */
+  onResize_() {
+    const triggered = this.triggered_;
+
+    // Stop animation right away.
+    if (this.runner_) {
+      this.runner_.cancel();
+      this.runner_ = null;
+    }
+
+    // Retstart the animation, but debounce to avoid re-starting it multiple
+    // times per restart.
+    this.triggered_ = triggered;
```

Comment:
I don't understand this assignment. Isn't that just `this.triggered_ = this.triggered_`?